packages/sku/lib/runBin.mjs

```diff
@@ -1,6 +1,9 @@
 // @ts-check
-const path = require('node:path');
-const spawn = require('cross-spawn');
+import path from 'node:path';
+import spawn from 'cross-spawn';
+import { createRequire } from 'node:module';
+
+const require = createRequire(import.meta.url);
```

Comment:
Was thinking that this might not work if the thing being required is ESM. I guess that's the entire reason why synchronous ESM require is such a big deal, otherwise everyone would just use this. We'll see later on whether this actually works, but this might be a situation we _do_ want to refactor as part of the ESM migration.

Reply:
I think once we're in the "bug resolution" phase of the beast it would be good to do small refactors and turn everything into async imports if possible.

Reply:
Currently, this is only requiring a `package.json` file.

Reply:
Oh yeah good point. Should hopefully be fine then? We'll find out I guess.

Reply:
> We'll find out I guess.

![what_could_go_wrong](https://github.com/user-attachments/assets/bb17bdad-cd1b-42d7-8817-dfb9102b7ab7)